### DISCUSSION TITLE
columnchart: bar-width

### DISCUSSION THREAD

**Discussion by mjhuesmann** [cetz-package/cetz-plot#58] [Q&A] [answered]
I tried to produce a columnchart of a binomial distribution. The bars have to have the width 1. Columnchart has the parameter bar-width, that defaults to 0.8. Changing the value to 1 or to 0.1 has no effect on the barwidth. What is wrong with my code?
Manfred

Code:
```
#import "@preview/cetz:0.2.2": chart, canvas
#let data3 = (
  (0, 0.002),
  (1, 0.001),
  (2, 0.002),
  (3, 0.003),
)
#canvas({
  //import cetz.draw: *
  // Your drawing code goes here
   chart.columnchart(
    size: (3, 2),
    bar-width: 0.8,
    x-inset: 0,
    y-tick-step: 0.001,
    y-format: "sci",
    data3)

})
```

**Comment by johannes-wolf** [comment, marked as answer]
This is a bug. You can use `cetz.draw.set-style(columnchart: (bar-width: ...))` as a workaround.

**Reply by mjhuesmann** [reply]
Thanks, a lot for the solution.
